Patch suggested by Manuel Teira to actually enable offscreen pixmap 
acceleration in Xw100. Value 16 is empirical, works well on hx4700,
but in case of issues, consider double it (other accelerated drivers
use bigger values than 16).

--- xorg-server-1.2.0/hw/kdrive/w100/ati_draw.c.org	2007-04-04 10:28:57.000000000 +0000
+++ xorg-server-1.2.0/hw/kdrive/w100/ati_draw.c	2007-04-06 14:43:40.000000000 +0000
@@ -433,6 +433,7 @@
      * or kaaPixmapUseScreen. But this is probably caused by some bug in this
      * driver...  */
     atis->kaa.flags |= KAA_OFFSCREEN_PIXMAPS;
+    atis->kaa.pitchAlign = 16;
     if (!kaaDrawInit(pScreen, &atis->kaa))
         return FALSE;
 
